### ISSUE TITLE
Style Guides

### ISSUE THREAD

**Issue by Sun-base** [zuri-training/anima_lib_team-91#21]
The style guide will be divided among you guys in this order.

Color System | Strokes & Shadows | Logo  -  Oluwaseun Lawal
Grid System | Checkboxes | Radio Buttons | Alerts  -  Elizabeth Odeleye
Typography | Spacing System  -  Justina Catherine
Button System | Icons  -  Nwokolo Paschal
Dropdown Menu | Input Fields  -  Alade Yunus

**Comment by Sun-base** [comment]
Everyone has completed their tasks, except for Alade Yunus whose didn't meet the deadline. The responsibility was later transferred to Kingsley who helped him out

**Comment by Sun-base** [comment]
Work completed. Link attached below
https://www.figma.com/file/7KC7ZQMjTujDu1qgyaUnxw/StyleFlow-Project?node-id=174%3A206

**Comment by AladeYunus** [comment]
> Everyone has completed their tasks, except for Alade Yunus whose didn't meet the deadline. The responsibility was later transferred to Kingsley who helped him out

REPLY:
I am sorry for missing the deadline set,

1. I had initially worked on the drop-downs at the beginning of the project and when I previewed the Figma file it was still the same as I left it - I have now updated it to match the font and size specified in the style guide.
2. As of the time I previewed the input field it font type and size did not correspond with the style guide - I have also updated that.

Thank you, my apologies once again to the team.

**Comment by Sun-base** [comment]
Owing to my experience as a graphics designer, I was able to create a minimalistic logo for the project. The logo's font is consolas and it was inspired by the title of our project - StyleFlow. Similarly, I worked on the color system. I chose Dark Orchid (#HB20BB) as our primary color and generated various tints and shade from the color for the purpose of our work. Also, I used Islamic Green as our secondary color and created a shade of white and black for neutral/background colors. We also had 4 semantic colors which depict error, warning, success and information respectively.

Finally, I prepared the stroke and shadow guide. The strokes ranged from 1px to 16px depending on where they will be adopted. The shadows vary from text shadow to line shadow and shape shadow.

These guides have made the coding of this project easier on the developers.
